File: AtomGraphics/android/atomgraphics-library/src/main/java/com/neo/atomgraphics/graphics/GraphicsLayerGCanvas.java
package com.neo.atomgraphics.graphics;

import com.neo.atomgraphics.node.Node;

public class GraphicsLayerGCanvas extends GraphicsLayer {

    public GraphicsLayerGCanvas(PlatformLayerTextureView platformLayer, Node rootNode) {
        super(platformLayer, rootNode);
    }

    @Override
    protected String layerType() {
        return "gcanvas";
    }
}

File: AtomGraphics/graphics/GraphicsLayerGCanvas.cpp
//
// Created by neo on 2018/7/13.
// Copyright (c) 2018 neo. All rights reserved.
//

#include "GraphicsLayerGCanvas.h"
#include "GraphicsLayerBackingStoreGCanvas.h"

namespace AtomGraphics {
    GraphicsLayerGCanvas::GraphicsLayerGCanvas(AtomGraphics::PlatformLayer *platformLayer, AtomGraphics::Node *rootNode)
            : GraphicsLayer(platformLayer, rootNode) {
        m_backingStore = new GraphicsLayerBackingStoreGCanvas(this);
    }
}
